src/core/json/path.cc

```diff
@@ -55,14 +55,19 @@ IndexExpr IndexExpr::Normalize(size_t array_len) const {
     return IndexExpr(1, 0);  // empty range.
 
   IndexExpr res = *this;
+  auto wrap = [=](int negative) {
```

Comment:
nit: you need only array_len for readability it's better to use var instead of =

Reply:
agree, will change in the next PR.